services/core/fuse/web-client/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
```

Comment:
And remove this (move the contents to the root level one if need be)

Reply:
removed